index.js

```diff
@@ -1,27 +1,30 @@
-const bodyParser = require("body-parser");
 const express = require("express");
 require("dotenv").config();
-const app = express();
-const mongoose = require("mongoose");
-const todoRoutes = require("./routes/todo.js");
+const scrapeController = require("./controllers/webscrapeController");
+const { getDateRange } = require("./models/dates");
 
-main()
-  .then((res) => console.log(res))
-  .catch((err) => console.log(err));
+const { startDate, endDate } = getDateRange();
+const mongoose = require("mongoose");
 
 async function main() {
-  await mongoose.connect(process.env.DB);
-}
+  try {
```

Comment:
why is all the api wrapped in try catch?
This becomes a problem if we don't depend on DB for api